src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Review/index.tsx

```diff
@@ -85,9 +86,11 @@ const ContractInteractionReview = ({ onClose, onPrev, tx }: Props): React.ReactE
     safeTxGas: manualSafeTxGas,
     manualGasPrice,
     manualGasLimit,
+    manualSafeNonce,
   })
 
-  const doExecute = isExecution && executionApproved
+  const canTxExecute = useCanTxExecute(false, manualSafeNonce)
+  const doExecute = canTxExecute && executionApproved
```

Comment:
This seems to be the same as in `ReviewConfirm#L130` but with a different name i.e. `doExecute` vs. `isExecution`. Can we use one or the other for the name to keep it consistent?